### ISSUE TITLE
Zobrazovanie článkov z blogu na webe pri projektoch

### ISSUE THREAD

**Issue by Teryii** [cesko-digital/web#594]
Ahojky,

pri zhŕňaní bilancie z predošlého roku kvôli narodeninám som si všimla, že hoci dávame tagy projektov ku všetkým blogovým článkom na blogu, články sa na webe pri projekte nezobrazujú všetky.

Je to tak naschvál? A nedalo by sa to nastaviť aspoň podľa najnovšieho dátumu, ak to vypisuje iba prvé tri? Pretože pri Nedlužím státu sú tam napríklad 3 články, ale ešte z prvého launchu, ktoré sú oproti 2. launchu už neaktuálne a hodili by sa skôr tie novšie. 😊

Pridávam aj image.

Dalo by sa s tým niečo urobiť? Poprípade si myslím, že netreba tam mať tie perexy, ktoré to zbytočne predlžujú, stačil by názov článku a odkaz. 😊 Aby sa tam toho zmestilo viac. 😁

Díky!

![cd-web-blog-sekcia-projekty](https://user-images.githubusercontent.com/90608819/168286479-c293d483-c8c9-4245-be7f-be26aa34ad04.png)

**Comment by zoul** [comment]
To chronologické pořadí byla chyba, díky, opraveno – teď jsou blog posty od nejnovějšího zpátky v čase. Kdybychom tam těch článků chtěli víc, bude to ještě relevantní, nebudou tam už ultrastaré věci? Nebo tam i ty ultrastaré věci dávají smysl?

**Comment by Teryii** [comment]
Jo, dávajú tam aj tie staré veci zmysel, často to viac uvádza ten projekt a pod., možno keby sa zobrazovalo 6 článkov, tak by to bolo akurát. 😊 Preto som navrhovala vymazať perex z toho zobrazenia, aby to zase nebolo kilometer dlhé. 😁
